src/tools/tools_description.rs

```diff
@@ -310,12 +320,19 @@ tools:
         type: "string"
         description: |
           Supports these commands:
-          navigate_to <url>
+          navigate_to <uri>
           screenshot
           html
           reload
+      - name: "tab"
+        type: "string"
+        description: "Tab where command will be executed."
+      - name: "screenshot"
+        type: "boolean"
+        description: "Optionally make a screenshot of a tab."
     parameters_required:
       - "command"
+      - "tab"
```

Comment:
Parameters are unclear.
1. Two ways to make a screenshot: "screenshot" command as in examples, and "screenshot" boolean parameter.
2. Tab is a string, a required parameter? How does it know what string to write there?

Reply:
screenshot command needed to monitor current state of the tab without reloading it